client/src/components/App.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+
+  }
+
+  render() {
+    return (
+      <div className="parent">
+        <table>
+          <tbody>
+            <h1>hello world</h1>
+          </tbody>
+        </table>
+        <button onClick={this.handleClick}>Reset Game</button>
```

Comment:
Everything looks good. Are you just using the 'Connect Four' mini app as a template that you will update later?

Reply:
Good call